### PR TITLE
lxqt-panel French Translation

### PR DESCRIPTION
**Pull request by VRad49**

### CODE REVIEW

File: panel/translations/lxqt-panel_fr_FR.ts [hunk 1]
```diff
@@ -49,7 +49,7 @@
     </message>
     <message>
         <source>Length:</source>
-        <translation type="vanished">Longueur :</translation>
+        <translation type="vanished">Longueur:</translation>
```
Comment (f2404) [inline panel/translations/lxqt-panel_fr_FR.ts]: You don't need to translate the "vanished" strings - they are not used anymore.

File: panel/translations/lxqt-panel_fr_FR.ts [hunk 2]
```diff
@@ -99,178 +99,178 @@
...
         <location filename="../config/configpaneldialog.ui" line="58"/>
         <source>Icon size:</source>
-        <translation type="unfinished"></translation>
+        <translation>Taille d'icône</translation>
     </message>
     <message>
         <location filename="../config/configpaneldialog.ui" line="65"/>
         <source>Length:</source>
-        <translation type="unfinished">Longueur :</translation>
+        <translation>Longueur :</translation>
```
Comment (f2404) [inline panel/translations/lxqt-panel_fr_FR.ts]: Why space before colon?

File: panel/translations/lxqt-panel_fr_FR.ts [hunk 3]
```diff
@@ -99,178 +99,178 @@
...
         <location filename="../config/configpaneldialog.ui" line="129"/>
         <source>px</source>
-        <translation type="unfinished">px</translation>
+        <translation>px</translation>
     </message>
     <message>
         <location filename="../config/configpaneldialog.ui" line="140"/>
         <source>Rows count:</source>
-        <translation type="unfinished"></translation>
+        <translation>Nombre de rangées</translation>
```
Comment (f2404) [inline panel/translations/lxqt-panel_fr_FR.ts]: Colon missing.

File: panel/translations/lxqt-panel_fr_FR.ts [hunk 4]
```diff
@@ -278,63 +278,63 @@
...
         <location filename="../lxqtpanel.cpp" line="970"/>
         <source>Panel</source>
-        <translation>Bloc-notes</translation>
+        <translation>Tableau de bord</translation>
     </message>
     <message>
         <location filename="../lxqtpanel.cpp" line="973"/>
         <source>Configure Panel...</source>
-        <translation type="unfinished"></translation>
+        <translation>Configurer le tableau de bord</translation>
```
Comment (f2404) [inline panel/translations/lxqt-panel_fr_FR.ts]: ... missing.

File: panel/translations/lxqt-panel_fr_FR.ts [hunk 5]
```diff
@@ -278,63 +278,63 @@
...
     </message>
     <message>
         <location filename="../lxqtpanel.cpp" line="978"/>
         <source>Add Panel Widgets...</source>
-        <translation type="unfinished"></translation>
+        <translation>Ajouter des widgets au tableau de bord</translation>
```
Comment (f2404) [inline panel/translations/lxqt-panel_fr_FR.ts]: ... missing.

File: panel/translations/lxqt-panel_fr_FR.ts [hunk 6]
```diff
@@ -278,63 +278,63 @@
...
     <message>
         <location filename="../lxqtpanel.cpp" line="991"/>
         <source>Remove Panel</source>
-        <translation type="unfinished"></translation>
+        <translation>Supprimer le tableau de bord</translation>
     </message>
     <message>
         <source>Configure panel...</source>
-        <translation type="vanished">Configurer le tableau de bord...</translation>
+        <translation>Configurer le tableau de bord...</translation>
```
Comment (f2404) [inline panel/translations/lxqt-panel_fr_FR.ts]: Are this one and the ones below still in use? Seems they are all "vanished".

**Comment by paulolieuthier** [comment]
Merged in c94919ea6f5df501d13cb4298a7d4401f70d4fc9. Thanks.

**Pull request closed without merging**